src/placeos-rest-api/utilities/current-user.cr

```diff
@@ -7,9 +7,20 @@ require "placeos-models/api_key"
 
 module PlaceOS::Api
   # Helper to grab user and authority from a request
+
+  enum Scope
```

Comment:
I think we should change this to

```
@[Flags]
enum Access
  Read  # 1
  Write # 2
end
```

which has implicit members `None` and `All`

Reply:
Whoops that shouldn't be there anymore, I added the Flags in models where this is actually defined

Reply:
ahh nice :)